lib/search_param_parser.rb

```diff
@@ -3,45 +3,36 @@
 module SearchParamParser
   def self.parse_movie_params(params)
     output = {
-      actor: params[:actor],
-      actor2: params[:actor2],
+      actor_name: params[:actor_name],
       year: nil,
-      exact_year: nil,
-      year_select: params[:year_select],
-      before_year: nil,
-      after_year: nil,
+      year_display: nil,
+      timeframe: params[:timeframe],
       genre: params[:genre],
       company: params[:company],
       mpaa_rating: params[:mpaa_rating],
       page: (params[:page].presence || 1),
-      sort_by: (params[:sort_by].presence || 'revenue')
+      sort_by: (params[:sort_by].presence || 'popularity')
```

Comment:
🤷 Popularity was the default in another part of the codebase, so I made this match.